app-admin/lsat/lsat-0.9.8.2.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils toolchain-funcs
+
+DESCRIPTION="The Linux Security Auditing Tool"
+HOMEPAGE="http://usat.sourceforge.net"
+SRC_URI="https://github.com/triode3/lsat/archive/v${PV}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="minimal"
+
+DEPEND="dev-lang/perl" # pod2man
+RDEPEND="
+	${DEPEND}
+	!minimal? (
+		app-portage/portage-utils
+		net-analyzer/nmap
+		sys-apps/iproute2
+		sys-apps/which
+		sys-process/lsof
+	)"
+
+PATCHES=(
+	"${FILESDIR}/${P}-gentoo.patch"
+	"${FILESDIR}/${P}-segfault-fix.patch"
```

Comment:
Are these patches still valid? Upstream hasn't fixed them since latest release?

Reply:
The latest version still segfaults without these patches. I actually had to slightly modify it to get it to build correctly (see below).